doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
I’m not picky either, but I can say that I’ve never regretted using a long identifier if it is just a tiny bit more expressive or easier to understand; muscle memory and/or autocomplete take care of that once you’re used to typing them.

`executionAsyncId()` and `triggerAsyncId()` sound fine to me, with or without the `current` prefix and a slight preference for having the `Async` middle bit.